lib/OktaAuth.ts

```diff
@@ -49,7 +49,7 @@ import {
   transactionStatus,
   resumeTransaction,
   transactionExists,
-  introspect,
+  introspectAuthn,
```

Comment:
looks like a breaking change, add to changelog?

Reply:
yes, I guess it would technically be a breaking change if anyone was importing { introspect } from `@okta/okta-auth-js`. Hopefully nobody is doing this and everyone is using `authClient.tx.introspect`. I'll add a note in changelog.